rct229/ruletest_engine/ruletest_jsons/section5/rule_5_36.json

```diff
@@ -0,0 +1,370 @@
+                                                        "id": "Surface 1",
+                                                        "adjacent_to": "EXTERIOR",
+                                                        "tilt": 30,
+                                                        "area": 92.90303999999999,
+                                                        "subsurfaces": [
+                                                            {
+                                                                "id": "Subsurface 1",
+                                                                "glazed_area": 4.6451519999999995,
+                                                                "opaque_area": 2.3225759999999998
+                                                            },
+                                                            {
+                                                                "id": "Subsurface 2",
+                                                                "glazed_area": 41.806368,
+                                                                "opaque_area": 4.6451519999999995
+                                                            }
+                                                        ]
+                                                    },
+                                                    {
+                                                        "id": "Surface 2",
+                                                        "adjacent_to": "EXTERIOR",
+                                                        "tilt": 50,
+                                                        "area": 185.80607999999998
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilation_air_conditioning_systems": [
+                                            {
+                                                "id": "HVAC 1",
+                                                "cooling_system": {
+                                                    "id": "Cooling System 1",
+                                                    "sensible_cool_capacity": 35170.0
+                                                },
+                                                "heating_system": {
+                                                    "id": "Heating System 1",
+                                                    "heat_capacity": 8792.133333333333
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    },
+    "rule-5-36-b": {
```

Comment:
The 5-36 logic is:
`if sum(subsurface_skylight_area_in_a_surface_b) / total_skylight_area_b  == sum(subsurface_skylight_area_in_a_surface_p) / total_skylight_area_p` then pass.
In this case, 5-36-b should pass.

To make failed case, one of the subsurface should move to Surface 2.